src/components/ItemEditorPage/TopPanel/TopPanel.tsx

```diff
@@ -3,13 +3,14 @@ import { Button, Loader } from 'decentraland-ui'
 import { t } from 'decentraland-dapps/dist/modules/translation/utils'
 import { locations } from 'routing/locations'
 import { Collection, CollectionType } from 'modules/collection/types'
-import { Curation, CurationStatus } from 'modules/curation/types'
+import { CollectionCuration, CurationStatus } from 'modules/collectionCuration/types'
 import { getCollectionType, hasReviews } from 'modules/collection/utils'
 import CollectionProvider from 'components/CollectionProvider'
 import JumpIn from 'components/JumpIn'
 import RejectionModal from './RejectionModal'
 import { RejectionType } from './RejectionModal/RejectionModal.types'
 import { ButtonType, Props, State } from './TopPanel.types'
+import { ItemCuration } from 'modules/itemCuration/types'
```

Comment:
can we move this import below `modules/collectionCuration/types`?

Reply:
For sure! moved